src/storage/src/error.rs

```diff
@@ -82,6 +83,30 @@ pub enum Error {
         source: object_store::Error,
     },
 
+    #[snafu(display(
+        "Fail to compress object by {}, path: {}, source: {}",
```

Comment:
nit: Based on my observation, @evenyag uses `Fail to`, and @v0y4g3r uses `Failed to`. Should we consider unifying the style?

Reply:
I don't have a strong preference about this, I usually follow the same message style in the error mod.